e2etest/e2etest_suite_test.go

```diff
@@ -167,7 +168,18 @@ var _ = ginkgo.BeforeSuite(func() {
 	framework.ExpectNoError(err)
 	v4Addresses := strings.Split(ipv4ForContainers, ",")
 	v6Addresses := strings.Split(ipv6ForContainers, ",")
-	frrContainers = setupContainers(v4Addresses, v6Addresses)
+	frrContainers, err = setupContainers(v4Addresses, v6Addresses)
+	framework.ExpectNoError(err)
+
+	// Allow the speaker nodes to reach the multi-hop network containers.
+	if containersNetwork != "host" {
+		speakerPods := getSpeakerPods(cs)
+		for _, pod := range speakerPods {
```

Comment:
Can we do this using nodes and containerExecutor? Kind nodes are containers after all, and this whole machinery will work only when using kind / docker. Is there a value in using the speakers?

I think using nodes will be more clear, related to what we want to do.

Reply:
That's what happening, added a comment that clarifies it - running over the speaker pods and creating executors from their node's name. Went that approach for if/when we start to create speakers only on specific nodes.